appveyor.yml

```diff
@@ -1,76 +0,0 @@
-version: 4.1.0.{build}
-clone_depth: 10
-install:
-build: off
-matrix:
-  fast_finish: true
-  exclude:
-    - platform: Ruby25-x64
-      LOCALE: ja-JP
-    - platform: Ruby24-x64
-      LOCALE: ja-JP
-
-# Ruby versions under test
-platform:
-  - Ruby26-x64
-  - Ruby25-x64
-  - Ruby24-x64
-  - Ruby23-x64
-
-# Note that locales are mainly code page changes and are not the FULL set of localization
-#  changes that happen when you install French/Germam/Japanese etc. Windows
-environment:
-  matrix:
-  - LOCALE: en-US
-  - LOCALE: ja-JP
```

Comment:
We've lost the Japanese test cells, which I think is ok for now? It might be nice to add French or German Windows at some point. Ruby tends to have more issues on Windows because it uses sets the external default encoding to the system's locale/code page instead of UTF-8.

Reply:
Yes, unfortunately i don't think github actions have windows runners with other locales already set.